README.md

```diff
@@ -70,7 +70,7 @@ Please refer to the [SNVPhyl documentation](https://snvphyl.readthedocs.io/en/la
 In order to run the pipeline with provided data, please run:
 
 ```
-nextflow run phac-nml/snvphylnfc -profile singularity --input https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv --refgenome https://raw.githubusercontent.com/phac-nml/snvphylnfc/main/assets/reference.fasta --outdir results
+nextflow run phac-nml/snvphylnfc -profile singularity --input https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv --refgenome https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/reference.fasta --outdir results
```

Comment:
I would probably leave the `README.md` and `assets/samplesheet.csv` as pointing to the `main` branch, but the tests can refer to the `dev` branch.

Reasoning being that we could then make updates to tests, samplesheet, etc in `dev`, and the docs on `main` for people trying out the pipeline would still all work with the released version of SNVPhyl.